auth/src/adapter/redis-message-bus-adapter.ts

```diff
@@ -0,0 +1,37 @@
+import Redis from 'redis';
+
+import { Logger } from '../api/services/logger';
+import { MessageBus } from '../api/interfaces/message-bus';
+
+export class RedisMessageBusAdapter extends MessageBus {
+    private messageBus: Redis.RedisClient;
+
+    public constructor() {
+        super();
+        this.init();
+    }
+
+    public async sendEvent(eventName: string, value: any): Promise<boolean> {
+        return await new Promise((resolve, reject) => {
+            this.messageBus.set(eventName, JSON.stringify(value), (error, result) => {
```

Comment:
You have to use redis streams: https://redis.io/commands/xadd

xadd logout * sessionId <the id>